conf/messages

```diff
@@ -274,6 +274,9 @@ assessmentCentre.analysisExercise.upload.tooBig=Your analysis exercise must be l
 assessmentCentre.analysisExercise.upload.wrongContentType=Your analysis exercise must be in the .doc or .docx format
 assessmentCentre.analysisExercise.upload.success=You''ve successfully submitted your analysis exercise.
 
+assessmentCentre.event.confirm.success=You''ve successfully confirmed for the event.
```

Comment:
This doesn't read quite right. Maybe something like "You've successfully confirmed acceptance for the event"